installer/packer/scripts/systemd/kov/kov_startup.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Kubernets on Vsphere
+Documentation=https://github.com/supervised-io/kov
+After=admiral.service
+Before=kov.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/bash /etc/vmware/kov/configure_kov.sh
```

Comment:
don't preface with /usr/bin/bash set the right permissions on the script file instead.
move this into ExecStartPre in kov.service and get rid of this unit